molgenis/molgenis-emx2
# chore(directory): Update collection categories

What are the main changes you did:
- Added new categories (autoimmune, cardiovascular, infectious, metabolic, nervous_system, oncology, population, and paediatrics) to the Categories Ontology table in the biobank-directory
- Delete three deprecated categories (paediatric_included, paediatric_only, and cancer)

how to test:
- Create new database with biobank_directory template 
- Check DirectoryOntologies => Categories and see that this contains 10 categories (autoimmune, cardiovascular, infectious, metabolic, nervous_system, oncology, population, covid10, rare_diseases and paediatrics)

todo:
- [na] updated docs in case of new feature
- [x] added/updated demo data
- [na] added/updated testplan to include a test for this fix, including ref to bug using # notation
